src/OrchardCore/OrchardCore.Localization.Core/DataAnnotations/DataAnnotationsDefaultErrorMessages.cs

```diff
@@ -1,118 +1,121 @@
+        public string CustomValidationAttributeMethodMustReturnValidationResult =>  S["The CustomValidationAttribute method '{0}' in type '{1}' must return System.ComponentModel.DataAnnotations.ValidationResult.  Use System.ComponentModel.DataAnnotations.ValidationResult.Success to represent success."];
 
-        public string CustomValidationAttributeMethodNotFound => S["The CustomValidationAttribute method '{0}' does not exist in type '{1}' or is not public and static."];
+        public string CustomValidationAttributeMethodNotFound =>  S["The CustomValidationAttribute method '{0}' does not exist in type '{1}' or is not public and static."];
 
-        public string CustomValidationAttributeMethodRequired => S["The CustomValidationAttribute.Method was not specified."];
+        public string CustomValidationAttributeMethodRequired =>  S["The CustomValidationAttribute.Method was not specified."];
 
-        public string CustomValidationAttributeMethodSignature => S["The CustomValidationAttribute method '{0}' in type '{1}' must match the expected signature: public static ValidationResult {0}(object value, ValidationContext context). The value can be strongly typed. The ValidationContext parameter is optional."];
+        public string CustomValidationAttributeMethodSignature =>  S["The CustomValidationAttribute method '{0}' in type '{1}' must match the expected signature: public static ValidationResult {0}(object value, ValidationContext context). The value can be strongly typed. The ValidationContext parameter is optional."];
 
-        public string CustomValidationAttributeTypeConversionFailed => S["Could not convert the value of type '{0}' to '{1}' as expected by method {2}.{3}."];
+        public string CustomValidationAttributeTypeConversionFailed =>  S["Could not convert the value of type '{0}' to '{1}' as expected by method {2}.{3}."];
 
-        public string CustomValidationAttributeTypeMustBePublic => S["The custom validation type '{0}' must be public."];
+        public string CustomValidationAttributeTypeMustBePublic =>  S["The custom validation type '{0}' must be public."];
 
-        public string CustomValidationAttributeValidationError => S["{0} is not valid."];
+        public string CustomValidationAttributeValidationError =>  S["{0} is not valid."];
 
-        public string CustomValidationAttributeValidatorTypeRequired => S["The CustomValidationAttribute.ValidatorType was not specified."];
+        public string CustomValidationAttributeValidatorTypeRequired =>  S["The CustomValidationAttribute.ValidatorType was not specified."];
 
-        public string DataTypeAttributeEmptyDataTypeString => S["The custom DataType string cannot be null or empty."];
+        public string DataTypeAttributeEmptyDataTypeString =>  S["The custom DataType string cannot be null or empty."];
```

Comment:
Extrace space added on every line before `S[]`

Reply:
Good catch, I will fix it now ..